vendor/github.com/jacobsa/gcloud/gcs/read.go

```diff
@@ -41,6 +42,16 @@ func (b *bucket) NewReader(
 	// In Google-internal bug 19718068, it was clarified that the intent is that
 	// each of the bucket and object names are encoded into a single path
 	// segment, as defined by RFC 3986.
+	
+
+	// Switching to Go Storage Client Library 
+	if true{
```

Comment:
Could you fork this repo and make changes in that so that we can ensure this repo does not break while we test these changes?

Reply:
Hey Avi! Actually I have made the PR on storagelibrary branch (created by Swetha) which is different than master branch. So I think it should not affect the main code in the master branch.

Reply:
My bad. Missed that.